cogs/commands/misc.py

```diff
@@ -143,6 +143,17 @@ async def chat(self, ctx: Context, message: str = None, gpt4: bool = False):
     async def gpt4(self, ctx: Context, message: str):
         await self.chat(ctx, message, True)
 
+    @commands.hybrid_command()
+    async def translate(
+        self, ctx: Context, source: str, target: str, message: clean_content
```

Comment:
add a help text to explain what source and target are (see [https://github.com/UWCS/apollo/blob/4d30f637eb5c0ba27180ef552ab5999c1a7c45f5/cogs/commands/say.py#L15](https://github.com/UWCS/apollo/blob/4d30f637eb5c0ba27180ef552ab5999c1a7c45f5/cogs/commands/say.py#L15))

Also might be an idea to set a deafult value of source to en or auto